packages/camera/camera_web/lib/src/camera_web.dart

```diff
@@ -451,23 +451,33 @@ class CameraPlugin extends CameraPlatform {
 
   @override
   Future<void> startVideoRecording(int cameraId, {Duration? maxVideoDuration}) {
+    return startVideoCapturing(
+        VideoCaptureOptions(cameraId, maxDuration: maxVideoDuration));
+  }
+
+  @override
+  Future<void> startVideoCapturing(VideoCaptureOptions options) {
+    if (options.streamCallback != null || options.streamOptions != null) {
+      throw UnimplementedError('Streaming is not currently supported on web');
```

Comment:
The unit tests should test that this throws as expected.

Reply:
Done